components/mpas-ocean/src/shared/mpas_ocn_eddy_parameterization_helpers.F

```diff
@@ -284,11 +282,11 @@ subroutine ocn_eddy_compute_mixed_layer_depth(statePool, forcingPool)
         !$omp end do
         !$omp end parallel
 
-        if (associated(landIceMask) ) then
+        if (associated(landIceDraft) ) then
           !$omp parallel
           !$omp do schedule(runtime)
           do iCell = 1, nCells
-             dThreshMLD(iCell) = dThreshMLD(iCell) - abs(landIceDraft(iCell))*landIceMask(iCell)
+             dThreshMLD(iCell) = dThreshMLD(iCell) - abs(landIceDraft(iCell))
           end do
```

Comment:
@vanroekel, this is the piece of code I'd like to get your input on.  It's the only thing that I think will lead to non-BFB results in fields that could feed back on model dynamics.

I think I masked this with the `landIceMask` before without really thinking about the consequences.  But the ice draft is smoothed near the ice-shelf calving front on both the iceward and seaward sides.  It seems like we should remove this ice draft (the amount that the sea surface is depressed by the weight of an ice shelf) even in open ocean areas where it is nonzero (maybe 10 cells seaward of an ice shelf at most).

I just wanted to check with you because you'll have the best sense of what fields we should be testing for changes as a result of modifying `dThreshMLD`.  Which parts of the eddy parameterizaiton are affected?

Reply:
@xylar thanks for the explanation on the change.  I agree that if we have an added pressure on the sea surface we should account for this and so it makes sense to remove that iceMask.

At minimum this will cause some changes in the submesoscale eddy parameterization, my feeling is it will strengthen as I'd expect MLD to deepen with this change where the ice pressure wasn't being removed previously.  So checking the MLEvelocity (normal and vertical) fields would be a good start.

If you have redi enabled, I'd expect very slight changes there as well.  For that one you could take a look at the temperature_hmixTendency (or salinity) variable, assuming the tracer budgets are on.  But again am expecting small changes.

Reply:
@vanroekel, thanks for your intuition on what will change.

> I'd expect MLD to deepen with this change

I could be wrong but I expect the opposite -- I think the MLD was artificially deep because the old code didn't account for the fact that much of that depth was actually because the sea surface was suppressed by land ice.

So it may be that we would expect a weakening of those parameterizations around ice-shelf fronts.  I'll take a look and see which it is.

Thanks again!

Reply:
Sorry you are right @xylar I thought this bit of code was correcting the pressure when finding the reference pressure level, but is a correction to the MLD itself.  Yes MLD will definitely shallow with this change.